dags/cluster/config/model.py

```diff
@@ -43,10 +59,18 @@ class ClusterConfig(BaseModel):
     # ---------------------------------------
     # Listings & Mappings
     # ---------------------------------------
-    fields_used: list[str]
-    fields_all: list[str]
-    mapping_source_ids_by_codes: dict[str, int]
-    mapping_acteur_type_ids_by_codes: dict[str, int]
+    # On fait la distinction entre les champs meta
+    # qu'on ne souhaite pas transformer
+    fields_used_meta: list[str]
+    # Et les champs data qui peuvent être transformés
+    # (ex: normalisation). Dans la validation de config
+    # on vient enrichir cette liste avec tous les champs
+    # sélectionnés par l'utilisateur du DAG Airflow
+    fields_used_data: list[str]
```

Comment:
Idem ici, on peut imaginer refléter le fait que ces champs s'apprêtent à recevoir une mise à jour 
```suggestion
    fields_ready_for_transformation: list[str]
```

On peut éviter une verbosité trop grand, mais "fields used data" n'a aucune signification à la première lecture

Reply:
OK je vais essayer de trouver quelque chose de mieux